projects/ng-aquila/src/dropdown/item/dropdown-item.ts

```diff
@@ -212,13 +219,22 @@ export class NxDropdownItemComponent implements Highlightable, OnDestroy, AfterV
   }
 
   private _showOrHideByFilter(search: string) {
-    const constraint = (this._dropdown as NxDropdownComponent).filterFn(search, this.viewValue);
+    const constraint =
```

Comment:
In the light of separation of concerns, wouldn't it be better to remove the dropdown dependency here all together and instead implement the logic in the dropdown?

Reply:
hi fabian, that is out of scope. this is only to remove dependency. required for angular 13.